Funcky.Money/MoneyEvaluationContext.cs

```diff
@@ -60,14 +60,9 @@ private Builder(Option<Currency> currency, Option<decimal> distributionUnit, Opt
         }
 
         public MoneyEvaluationContext Build()
-        {
-            if (CompatibleRounding().Match(none: false, some: Negate))
-            {
-                throw new IncompatibleRoundingException($"The roundingStrategy {_roundingStrategy} is incompatible with the smallest possible distribution unit {_distributionUnit}.");
-            }
-
-            return CreateContext();
-        }
+            => CompatibleRounding().Match(none: false, some: Not<bool>(Identity))
+                ? throw new IncompatibleRoundingException($"The roundingStrategy {_roundingStrategy} is incompatible with the smallest possible distribution unit {_distributionUnit}.")
```

Comment:
why is the r of rounding strategy lower case?

Reply:
(and no need for a trailing dot)

Reply:
Fixed: 9755c05be57f836d703ebf1ebfa086d14abdc938